examples/create-mcs-db-cluster-with-shards/provider.tf

```diff
@@ -0,0 +1,15 @@
+provider "mcs" {
+    username          = "user@mail.ru"
+    password          = "password"
+    project_id        = "project_id"
+    auth_url          = "https://infra.mail.ru/identity/v3/"
+}
+
+provider "openstack" {
+    user_name        = "user@mail.ru"
+    password         = "password"
+    tenant_id        = "project_id"
+    user_domain_name = "users"
```

Comment:
how does mcs provider work w/o this?

Reply:
The same field (user_domain_name) is not present in working examples for kubernetes cluster